t-update-filter-behavior.html

```diff
@@ -188,7 +179,7 @@
     },
 
     _executeFunctionByName: function (functionName, context /*, args */) {
-        var args = [].slice.call(arguments).splice(2);
```

Comment:
Do you really need `splice` here?. Slicing the arguments can be handled in `slice` call itself.